pkgs/by-name/ty/typst/package.nix

```diff
@@ -60,7 +61,11 @@ rustPlatform.buildRustPackage (finalAttrs: {
   versionCheckProgramArg = [ "--version" ];
   doInstallCheck = true;
 
-  passthru.updateScript = nix-update-script { };
+  passthru = {
+    updateScript = nix-update-script { };
+    typstPackages = callPackage ./typst-packages.nix { };
```

Comment:
Why not just `packages` like e.g. Python has?

Reply:
Good idea.